lib/commands/create.js

```diff
@@ -11,21 +11,23 @@ module.exports = function (program) {
   program
     .command('create <shortname-without-spaces>')
     .description(stripIndent`
-      Create an Apostrophe project (using the A3 boilerplate by default).
+      Create an Apostrophe project (using the A3 Essentials starter kit by default).
       Example: \`apos create my-website\`
     `)
     .option('--a2', 'Use the Apostrophe 2 boilerplate')
-    .option('--starter <url>', 'Use a specific git repository to use as the project starter')
+    .option('--starter <url>', 'Use a specific git repository to use as the project starter.\n You can also use the short name of any Apostrophe starter kit, e.g. "ecommerce"')
     .action(async function(shortName, options) {
-      if (!options.starter && options.boilerplate) {
-        // Support the old boilerplate flag to be nice.
-        options.starter = options.boilerplate;
-      }
+      // If options.starter is undefined, use config.A3_BOILERPLATE
+      const input = options.starter ? options.starter : config.A3_BOILERPLATE;
 
-      const boilerplateUrl = options.starter ? options.starter
-        : options.a2 ? config.A2_BOILERPLATE : config.A3_BOILERPLATE;
+      // Format the URL according to your rules
+      const boilerplateUrl = /^(http|https):\/\//i.test(input)
```

Comment:
Better to check for anything with a protocol — it could be an ssh github URL for instance.

Reply:
Modified ternary to include ssh URL.